source/examples/examples/tutorials.md

```diff
@@ -5,16 +5,25 @@ containerClass: examples
 
 # Best Practices
 
-In this tutorial, we'll focus on writing and organizing tests for a real application.
+This video by the creator of Cypress Brian Mann shows what we consider the best practices for writing tests for a realistic application.
```

Comment:
This isn't really a tutorial though. It's a talk. I would remove this section, we have this linked in other places. This page is for tutorials to walk someone through testing something from start to finish. 

The other section looks good now though - having the list above where the other tutorial starts.

Reply:
I would not remove it just yet, this is an extremely popular video, I would punt on the removal

Reply:
Just because it's popular doesn't mean it belongs in this section of our docs.

Reply:
:) probably, but maybe we can handle it separately from adding more video links

Reply:
I have opened a separate issue to move it https://github.com/cypress-io/cypress-documentation/issues/2912